package.json

```diff
@@ -27,7 +27,9 @@
     "lodash": "^4.17.11",
     "popper.js": "^1.16.0",
     "react": "^16.8.6",
+    "react-bootstrap": "^1.6.0",
```

Comment:
do we need react-bootstrap? bootstrap is already included as a dep?

Reply:
The button and form I used in the login were from this package, and I think it's separate.